components/d2l-work-to-do/d2l-work-to-do-activity-list-item-basic.js

```diff
@@ -208,9 +211,7 @@ class ActivityListItemBasic extends ListItemLinkMixin(SkeletonMixin(EntityMixinL
 						${this._name}
 					</div>
 					<div class=${classMap(secondaryClasses)} slot="supporting-info">
-						${dateTemplate}
-						${separatorTemplate}
-						${this._orgName || this._orgCode}
+						${supportingInfoTemplate([dateTemplate, (this._orgName || this._orgCode), this._type === ActivityAllowList.userCourseOfferingActivity.type ? this._type : null])}
```

Comment:
Swap the other side for the pre-translated type, and give 'er some space.

```suggestion
						${supportingInfoTemplate([
						  dateTemplate,
						  this._orgName || this._orgCode,
						  this._activityProperties.type === ActivityAllowList.userCourseOfferingActivity.type && this._type
						])}
```

Reply:
oops - totally forgot that `this._type` has itself been translated